src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -24,13 +18,32 @@ export interface DropdownMenuProps {
     className?: string;
 }
 
-export const DropdownMenu = ({ dropdownMenuItems, username, className }: DropdownMenuProps) => {
+export const DropdownMenu = ({ dropdownMenuItems, className }: DropdownMenuProps) => {
     const [isOpen, setIsOpen] = useState(false);
+    const wixApi = useContext(WixAPIContext);
+    const [username, setUsername] = useState<string | null>(null);
```

Comment:
Can we use an empty string `''` as the default value instead of `null`?

Reply:
I actually did that on purpose, I think that null is the right thing to use here since it clearly indicates an uninitialized state, whereas an empty string could imply a valid but empty value.